src/hssm/hssm.py

```diff
@@ -404,6 +404,7 @@ def __init__(
             self.model, self._parent_param, self.response_c, self.response_str
         )
         self.set_alias(self._aliases)
+        # _logger.info(self.pymc_model.initial_point())
```

Comment:
Stylistically eventually yes, but rn, I think it can sometimes still help future PRs that interact with this code. Here I literally have the next PR that I need to work on in mind. So in general agree, but let's skip here :)